src/reducers/entities/posts.js

```diff
@@ -314,6 +314,65 @@ function openGraph(state = {}, action) {
     }
 }
 
+function messagesHistory(state = {}, action) {
+    const index = {};
+    index[Posts.MESSAGE_TYPES.POST] = -1;
+    index[Posts.MESSAGE_TYPES.COMMENT] = -1;
+
+    const nextIndex = state.index ? {...state.index} : index;
+    let nextMessages = state.messages ? [...state.messages] : [];
```

Comment:
Just a small note as I don't think this is an issue in this case, but every time any action dispatches this code will run even if the reducer doesn't need to do anything. It's best to keep the setup outside the switch statement to a minimum

I don't think it's a problem here because the setup should have negligible effect but just something to keep in mind

Reply:
The approach here was to reduce the quantity of repeated code, but I can change it.

Reply:
No change required just wanted to give a heads-up